src/main/java/fr/ospiea/oscra/employee/controller/EmployeeController.java

```diff
@@ -0,0 +1,47 @@
+package fr.ospiea.oscra.employee.controller;
+
+import fr.ospiea.oscra.employee.object.Employee;
+import fr.ospiea.oscra.employee.service.EmployeeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * Created by fqin on 18/09/2016.
+ */
+@RestController
+@RequestMapping("/api/employees")
```

Comment:
not really. You should search the solution that reply to every controller instead of copying '/api' everywhere.
The solution: add 
# server.contextPath=/api 

in the application.properties